cluster/manifests/01-admission-control/teapot.yaml

```diff
@@ -167,7 +167,7 @@ webhooks:
     sideEffects: "NoneOnDryRun"
     matchPolicy: Equivalent
     rules:
-      - operations: [ "CREATE", "UPDATE", "DELETE" ]
+      - operations: [ "DELETE" ]
```

Comment:
I believe that's not necessary because of https://github.bus.zalan.do/teapot/admission-controller/blob/master/pkg/admitters/serviceaccount/admit_serviceaccount.go#L39 but better be safe here.